themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -281,9 +400,47 @@ const cloudfrontDistribution = new aws.cloudfront.Distribution(
 );
 ```
 
+{{% /choosable %}}
+
+{{% choosable cloud gcp %}}
+
+Next, we want to front our Cloud Storage Bucket with a [Load Balancer](https://cloud.google.com/load-balancing/docs/https) and enable its [CDN capabilities](https://cloud.google.com/cdn/docs/overview). Add the following additional code to your `index.ts` file in order to create a GCP Load Balancer, Public IP address and URL Map to your GCP storage bucket.
+
+```typescript
+const backendBucket = new gcp.compute.BackendBucket("backend-bucket", {
+    bucketName: bucket.name,
+    enableCdn: true,
+});
+
+// Provision a global IP address for the CDN.
+const ip = new gcp.compute.GlobalAddress("ip", {});
```

Comment:
There's no need to pass `{}` for these as they're optional (and it's more Pulumi-idiomatic to leave them out):

```suggestion
const ip = new gcp.compute.GlobalAddress("ip");
```

Reply:
Fixed in 1e5040bd